### PR TITLE
refactor: adjust all dialogs to use oak components

### PR DESCRIPTION
**Pull request by tomwisecodes**
## Description
 
changes text on share page
adjusts all dialogs to use oak components for styling

Follow up discussion. 
Should we move away from radix to manage the dialog? 

solves:
https://www.notion.so/oaknationalacademy/Feedback-survey-scrollbar-modal-size-11426cc4e1b180aea716fa5668078d1f
https://www.notion.so/oaknationalacademy/Share-disclaimer-copy-update-9b8673eac6cb440bb314223547df97d4
https://www.notion.so/oaknationalacademy/Wonky-Spinner-13026cc4e1b1802ebb4ce27a11e012ee

### CODE REVIEW

File: apps/nextjs/src/components/AppComponents/Chat/Chat/types.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 export type DialogTypes =
   | ""
```
Comment (mantagen) [inline apps/nextjs/src/components/AppComponents/Chat/Chat/types.ts:2]: what's that dialog type?

File: apps/nextjs/src/components/DialogControl/DialogContents.tsx [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
 import EndOfLessonFeedback from "./ContentOptions/EndOfLessonFeedback";
 import ReportContentDialog from "./ContentOptions/ReportContentDialog";
 import ShareChatDialog from "./ContentOptions/ShareChatDialog";
-import {
-  dialogContent,
-  dialogContentInner,
-  dialogOverlay,
-} from "./dialogStyles";
+
+const dialogTitlesAndIcons: Record<
```
Comment (mantagen) [inline apps/nextjs/src/components/DialogControl/DialogContents.tsx:17]: Was gonna comment here but you already made the change!

**Review by mantagen — APPROVED** [review]
Code looks good!

**Review by mikeritson-oak — CHANGES_REQUESTED** [review]
I’ve added some testing notes to the top of the feedback and share modal tickets in Notion.

**Review by mikeritson-oak — CHANGES_REQUESTED** [review]
Rechecked for updates, mostly look OK but two minor amendments needed in the share modal. Also, some updates were also noted in the feedback Notion ticket.

**Review by mikeritson-oak — APPROVED** [review]
All looking good for a release to production now. Minor issues or design changes can be picked up in later tickets. 👍

**Pull request merged**